### DISCUSSION TITLE
boost 1.8.6 安装失败 error: package(boost): links not found!

### DISCUSSION THREAD

**Discussion by mengxinxiaoqiang** [xmake-io/xmake#5734] [Q&A]
![CNCJA NS2YF_PA(IBH6Y$F5](https://github.com/user-attachments/assets/2a0b76e0-77df-49ed-9959-7d35e552d23c)
XMake版本是DEV最新

**Comment by star-hengxing** [comment]
看一下 xmake.lua 配置

不过在 Windows 下大概是这个问题 https://github.com/xmake-io/xmake-repo/issues/4503 ，只能禁用 debug/asan 或者等这个 https://github.com/xmake-io/xmake-repo/issues/5437 完成
